docs/content/references/ts-sdk/dapp-kit/wallet-hooks/useAccounts.mdx

```diff
@@ -50,9 +17,7 @@ const UseAccountsExample = withProviders(() => {
             </ul>
         </div>
     );
-});
-
-render(<UseAccountsExample/>)
+}
```

Comment:
Unrelated to your PR technically, but could you remove the `-` in the `li`. It also happens in `useWallets` .

![image](https://github.com/user-attachments/assets/3e1fa27c-be3e-4dce-a1ae-88282d2f214d)

Reply:
Fixed in e01cc28